packages/react-button/src/components/Button/useButton.ts

```diff
@@ -1,11 +1,21 @@
 import { ButtonProps, ButtonSlots, ButtonSlotProps } from './Button.types';
 import { mergeProps } from '../../utils/mergeProps';
 import { ComposePreparedOptions } from '@fluentui/react-compose';
+import { tokensToStyleObject } from '@fluentui/react-theme-provider';
 
 /**
  * The useButton hook processes the Button component props and returns
  * state, slots, and slotProps for consumption by the component.
  * @param props
  */
-export const useButton = (props: ButtonProps, options: ComposePreparedOptions) =>
-  mergeProps<ButtonProps, ButtonSlots, ButtonSlotProps>(props, options);
+export const useButton = (props: ButtonProps, options: ComposePreparedOptions) => {
+  const { tokens, ...state } = props;
+
+  // This this go into mergeProps?
+  if (tokens) {
+    // tslint:disable-next-line: no-any
+    state.style = { ...props.style, ...tokensToStyleObject(tokens as Record<string, any>, '--button') };
+  }
```

Comment:
I don't know if it should go into `mergeProps` but maybe this whole thing could be the utility that you're importing instead of having to manually spread `...props.style` here.

```suggestion
  const state = {
    ...props,
    style: tokensToStyleObject(props, '--button')
  };